_posts/en/posts/2023-05-21-release-25.0.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Bitcoin Core 25.0 released
+name: blog-release-25.0
+id: en-blog-release-25.0
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement. English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 25.0 is now available.
+---
+Bitcoin Core version 25.0 is now available for [download][download
+page].  See the [release notes][] for more information about the many
+new features and bug fixes in this release.
+
+If you have any questions, please stop by the #bitcoin IRC chatroom
+([IRC][irc], [web][web irc]) and we’ll do our best to help you.
+
+[release notes]: /en/releases/23.1/
```

Comment:
Should this be 25.0 or you want to maintain 23.1?
`/en/releases/23.1/`

Reply:
Thanks. Fixed.